### ISSUE TITLE
hide default_view?

### ISSUE THREAD

**Issue by AnderssonPeter** [home-assistant/home-assistant-polymer#245]
Is there some way to hide `default_view` (HOME).

i have tried both
```
default_view:
  view: no
  control: hidden
```
and
```
homeassistant:
  customize:
    group.default_view:
      hidden: yes
```
none of them work.

**Comment by andrey-git** [comment]
Why do you want to hide default view?

**Comment by AnderssonPeter** [comment]
I have put each room in a group, with a icon in the menu and I don't have anything to put in the default view, so the idea was to hide it.

My first idea was to put all groups in the default view and the menu but it seems that's not possible.

**Comment by andrey-git** [comment]
Since 0.41 you can change the name and the icon of the `default_view`. Does this solve your usecase?

You must use
```
default_view:
  view: yes
```
to do so

**Comment by AnderssonPeter** [comment]
That should work ill upgrade soon!

**Comment by andrey-git** [comment]
Turns out last fronted changes weren't included in the last release :(

**Comment by balloob** [comment]
my bad

**Comment by AnderssonPeter** [comment]
Np, Could you guys update here when the change is published?

**Comment by balloob** [comment]
Will be published in 0.42.
